### ISSUE TITLE
Old packets no longer supported?

### ISSUE THREAD

**Issue by tlacson7** [HerculesWS/Hercules#2905]
**Describe the bug**
I was trying to try older packet versions listed below but it gives error during compile. Thanks for checking. 
`
// server->client protocol version
//        0 - pre-?
//        1 - ?                    - 0x196
//        2 - ?                    - 0x78, 0x79
//        3 - ?                    - 0x1c8, 0x1c9, 0x1de
//        4 - ?                    - 0x1d7, 0x1d8, 0x1d9, 0x1da
//        5 - 2003-12-18aSakexe+   - 0x1ee, 0x1ef, 0x1f0, ?0x1c4, 0x1c5?
//        6 - 2004-03-02aSakexe+   - 0x1f4, 0x1f5
//        7 - 2005-04-11aSakexe+   - 0x229, 0x22a, 0x22b, 0x22c
// 20061023 - 2006-10-23aSakexe+   - 0x6b, 0x6d
// 20070521 - 2007-05-21aSakexe+   - 0x283
// 20070821 - 2007-08-21aSakexe+   - 0x2c5
// 20070918 - 2007-09-18aSakexe+   - 0x2d7, 0x2d9, 0x2da
// 20071106 - 2007-11-06aSakexe+   - 0x78, 0x7c, 0x22c
// 20080102 - 2008-01-02aSakexe+   - 0x2ec, 0x2ed , 0x2ee
// 20081126 - 2008-11-26aSakexe+   - 0x1a2

#ifndef PACKETVER
	#define PACKETVER 20061023
#endif // PACKETVER
`

**System specs (please complete the following information):**
 - OS: Centos 7x64
 - Hercules Version [ v2020 - latest]
 - Mode: [pre-renewal?]
 - Packet version: [e.g. 20061023]
 - Client type: [main]

**Comment by 4144** [comment]
yes sadly some packet versions between near 2006-2009 may not compile.
at this packet versions gravity used totally random packets and packet sizes. it's hard to validate each of this packets.
